### PR TITLE
Fixed Products Export to work

### PR DESCRIPTION
**Pull request by Orrison**

### CODE REVIEW

**Comment by skylerkatz** [comment]
@Orrison it looks you made updates to the projects module on the same branch you had previously made updates to the events module that were already merged in.

We want this PR to only include the products module work.

**Comment by Orrison** [comment]
Yeah sorry, I am working with Chris to correct that.

File: Exports/_export/_modules/custom-type/products.php [hunk 1]
```diff
@@ -12,115 +12,121 @@
...
 		$out = str_replace('&amp;','&',$out);
 		$out = '"' . $out . '"';
+
 		return $out;
 	}
 
+	$filename = getSiteId() . '_' . 'products' . 'Export' . date('M') . '_' . date('d') . '_' . date('Y');
+	$howmany = 10000; // Set to number of products in the module
+
+	// Header
```
Comment (skylerkatz) [inline Exports/_export/_modules/custom-type/products.php]: _headers_ only need to be set once, but when they are inside of the `processItem` function they will get set multiple times.  I would put the headers back where they were previously, or right before you echo out the content near the bottom of the page.

File: Exports/_export/_modules/custom-type/products.php [hunk 2]
```diff
@@ -12,115 +12,121 @@
...
 	$get_products = '';
-	for($i=1; $i<=$batch_count; $i++){
+
+	for ($i=1; $i<=$batch_count; $i++) {
 
 		$this_howmany = strval($batch_length);
 		$this_offset = strval(($batch_length * ($i-1)));
 
 		$get_products .=
 		getContent(
```
Comment (skylerkatz) [inline Exports/_export/_modules/custom-type/products.php]: **Optional** - Convert this `getContent()` call to use the _json_ version.

**Pull request closed without merging**